packages/pn-personagiuridica-webapp/src/redux/delegation/types.ts

```diff
@@ -0,0 +1,79 @@
+import { RecipientType } from '@pagopa-pn/pn-commons';
```

Comment:
Pay attention. This models are already defined. They were grouped in another file during the development of the delegation part